pwa/src/components/common/elementCreationNew.tsx

```diff
@@ -72,18 +72,18 @@ const ElementCreationNew: React.FC<ElementCreationNewProps> = ({id, label, data}
       }
 
       <div className="row">
-        <h5>Create {label}</h5>
+        <h5>Add {label}</h5>
```

Comment:
Je bent hier aan het terugdraaien wat in [CON-2194](https://conduction.atlassian.net/browse/CON-2194) is opgepakt. Het juiste label hier is _Create_

Reply:
Zelfde geldt voor andere plekken waar je Create => Add hebt veranderd

Reply:
yes dat ga ik doen